OloEngine/src/OloEngine/Serialization/MeshBinarySerializer.cpp

```
@@ -0,0 +1,1543 @@
+#include "OloEngine/Renderer/MeshSource.h"
+#include "OloEngine/Renderer/MeshOptimization.h"
+#include "OloEngine/Renderer/Vertex.h"
+#include "OloEngine/Animation/Skeleton.h"
+#include "OloEngine/Animation/MorphTargets/MorphTarget.h"
+#include "OloEngine/Animation/MorphTargets/MorphTargetSet.h"
+#include "OloEngine/Animation/AnimationClip.h"
+#include "OloEngine/Debug/Instrumentor.h"
+
+#include <zlib.h>
+
+#include <cmath>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)